qhub_ops/template/{{ cookiecutter.repo_directory }}/infrastructure/dask-gateway.yaml

```diff
@@ -14,6 +14,7 @@ gateway:
       def worker_profile(options):
            config = profiles[options.profile]
            config['worker_cmd'] = '/opt/conda-run'
+           config['scheduler_cmd'] = '/opt/conda-run'
```

Comment:
We need to rename the `conda-run` script to something like `conda-run-worker` and create another script `conda-run-scheduler` reason being is that at the moment `conda-run` only runs the dask-gateway-worker see https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/image/dask-worker/postBuild#L10.

Reply:
Done, thanks!